src/test/java/Mona/ParserTest.java

```diff
@@ -0,0 +1,19 @@
+package mona;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+class ParserTest {
+
+    @Test
+    public void testGetCommand() throws MonaException {
```

Comment:
Maybe it could be better to extend this test to various forms of invalid inputs too, such as event commands without a "/from" and "/to", this could possibly lead to a greater coverage of the Parser class.